### PR TITLE
Make podman service log events

### LINKED ISSUE #8390: [RFE] podman system service: set default log level to info
/kind feature

I would love `podman system service` to be more verbose by default. At the moment it does not really print anything but I want to _see_ that it's alive, especially when throwing requests at it.

Setting the default log level to info could be a good first. Once done, I would love each request to be printed on info as well.

It may come in handy for future maintenance and debugging. The logs would now indicate that something's happening and also the order of requests etc.

@containers/podman-maintainers @WDYT?

### PR DESCRIPTION
**Pull request by jwhonce**
* Log endpoint calls at level Info
* Ensure API server started at level Info

Fixes #8390

Signed-off-by: Jhon Honce <jhonce@redhat.com>

### CODE REVIEW

File: cmd/podman/system/service_abi.go [hunk 1]
```diff
@@ -21,6 +21,11 @@ func restService(opts entities.ServiceOptions, flags *pflag.FlagSet, cfg *entiti
 		err      error
 	)
 
+	if logrus.GetLevel() < logrus.InfoLevel {
+		logrus.Debugf("Logging level upgraded from %q to %q", logrus.GetLevel().String(), logrus.InfoLevel.String())
+		logrus.SetLevel(logrus.InfoLevel)
+	}
+
```
Comment (TomSweeneyRedHat) [inline cmd/podman/system/service_abi.go:28]: I'm not sure where, but this feels like it should be documented somewhere.  I can see some admin trying to reset the level and having it go back up.

Comment (jwhonce) [reply cmd/podman/system/service_abi.go:28]: Added to man page.

File: docs/source/markdown/podman-system-service.1.md [hunk 2]
```diff
@@ -17,12 +17,14 @@ The REST API provided by **podman system service** is split into two parts: a co
 Documentation for the latter is available at *https://docs.podman.io/en/latest/_static/api.html*.
 Both APIs are versioned, but the server will not reject requests with an unsupported version set.
 
+Note: To provide additional information about remote requests **podman system service** always updates the **--log-level** to at least *info*.
```
Comment (mheon) [inline docs/source/markdown/podman-system-service.1.md:20]: "updates the log-level option to at least *info*"

**Comment by mheon** [comment]
One documentation nit, otherwise LGTM

**Review by vrothberg — COMMENTED** [review]
LGTM

**Comment by TomSweeneyRedHat** [comment]
LGTM
assuming happy tests

File: docs/source/markdown/podman-system-service.1.md [hunk 3]
```diff
@@ -17,12 +17,14 @@ The REST API provided by **podman system service** is split into two parts: a co
 Documentation for the latter is available at *https://docs.podman.io/en/latest/_static/api.html*.
 Both APIs are versioned, but the server will not reject requests with an unsupported version set.
 
+Note: To provide additional information about remote requests **podman system service** always updates the log-level option to at least *info*.
```
Comment (rhatdan) [inline docs/source/markdown/podman-system-service.1.md:20]: I think this should be a recommends and allow the user to flexibility to set it anyway they want.

**Comment by rhatdan** [comment]
Lets change the call in the service file to hard code --log-level=info.

**Review by vrothberg — COMMENTED** [review]
LGTM

Could we add another note in the `--help` text? Would be nice to mention that most users should run it with `--log-level=info`.

File: contrib/systemd/system/podman.service [hunk 4]
```diff
@@ -8,4 +8,5 @@ StartLimitIntervalSec=0
 [Service]
 Type=notify
 KillMode=process
-ExecStart=/usr/bin/podman system service
+Environment=LOGGING="--log-level=info"
```
Comment (rhatdan) [inline contrib/systemd/system/podman.service:11]: Why did you do it this way as opposed to just setting it in the ExecStart line?

Comment (jwhonce) [reply contrib/systemd/system/podman.service:11]: @rhatdan I can see the logging level easily and if desired change to default by commenting out one line.  My user podman.service has a whole list of Environment lines for debugging etc.

Comment (rhatdan) [reply contrib/systemd/system/podman.service:11]: SGTM

**Pull request merged**